src/core/packages/chrome/browser-internal/src/project_navigation/project_navigation_service.ts

```diff
@@ -90,6 +86,7 @@ export class ProjectNavigationService {
   private readonly nextSolutionNavDefinitionId$ = new BehaviorSubject<SolutionId | null>(null);
   // The active solution navigation definition id that has been initiated and is currently active
   private readonly activeSolutionNavDefinitionId$ = new BehaviorSubject<SolutionId | null>(null);
+  private readonly activeDataTestSubj$ = new BehaviorSubject<string | undefined>(undefined);
```

Comment:
I'd love to also get rid of this, but this would require touching tests and pinging more teams fore review. maybe later